src/Data/Currency/Currencies.hs

```diff
@@ -65,8 +66,11 @@ data CLP = CLP deriving (Show, Read, Eq)
 
 -- | European Union Euro
 data EUR = EUR deriving (Show, Read, Eq)
+--
+-- | British Poung Sterling
```

Comment:
Poung -> Pound

Reply:
Oh my goodness. I am useless without a compiler. Yes, I'll fix that. Thanks for catching it.